lib/mat8bit.gi

```diff
@@ -1004,7 +1004,7 @@ InstallMethod(SemiEchelonMatTransformationDestructive,
 ##
 
 InstallMethod(TriangulizeMat,
-        "kernel method for plain list of GF2 vectors",
+        "kernel method for plain list of GF2 vectors", # TODO not sure this is accurate
```

Comment:
This is definitely wrong, likely copy & past. From the C source comment:
`Method for TriangulizeMat for plain lists of 8 bit vectors`
So perhaps use this:
```suggestion
        "kernel method for plain list of 8 bit vectors",
```
The same problem then affects other methods, e.g. `DeterminantMatDestructive` below. Also, the comment for this method here in line 1003 refers to GF2, and so on.